### DISCUSSION TITLE
Question: can this lib support arm64 misp device ?

### DISCUSSION THREAD

**Discussion by fghjhuang** [pion/webrtc#1854] [Q&A] [answered]
i want to make my mt7688 board to use webrtc with a usb cam to see

**Comment by Sean-Der** [comment, marked as answer]
I believe so! I haven't done it myself but searching for "mt7688 golang" returns [linkit-smart-7688-golang](https://github.com/appleboy/linkit-smart-7688-golang)

[this](https://gist.github.com/asukakenji/f15ba7e588ac42795f421b48b8aede63) is a list of all OSes and architectures supported by Golang today.
